src/java/org/apache/cassandra/db/SystemKeyspace.java

```diff
@@ -141,6 +141,7 @@ private SystemKeyspace()
               + "version int,"
               + "PRIMARY KEY ((id)))")
               .partitioner(new LocalPartitioner(TimeUUIDType.instance))
+              .memtable(MemtableParams.DEFAULT)
```

Comment:
Do we need this? `MemtableParams.DEFAULT` is already the default

Reply:
Removed